conf.py

```diff
@@ -44,12 +54,19 @@
         }
 confname = 'unconfigured'
 
+def validate(condition, errmsg):
+	"""Function to wrap validation for use in validateConf"""
```

Comment:
"wrap validation" is a bit vague: try something like `"""Convenience function to validate conditions in validateConf().""""`